framer/Gestures.coffee

```diff
@@ -1,5 +1,6 @@
 # Gesture events
 Gestures = {}
+Gestures._prefix = "gesture:"
```

Comment:
Well it's for internal use only. It's how I distinguish between Events.* and Gestures.*.